sample-compose/src/androidTest/java/com/airbnb/lottie/samples/InfiniteAnimationTest.kt

```diff
@@ -23,7 +24,7 @@ import org.junit.Test
 class InfiniteAnimationTest {
 
     @get:Rule
-    val composeTestRule = createAndroidComposeRule(ComposeActivity::class.java)
+    val composeTestRule = createAndroidComposeRule(ComponentActivity::class.java)
```

Comment:
These tests were failing for me, with an error saying `setContent` can't be called twice. 
I noticed you're overwriting Content, so I used empty _generic_ activity